.github/workflows/i18n_server_list_update.yaml

```diff
@@ -21,6 +21,11 @@ jobs:
         uses: actions/checkout@v4
         with:
           submodules: "true"
+      - uses: actions/setup-python@v5
```

Comment:
Using v5 directly, as there's another PR trying to update all the workflows #9520